### PR TITLE
Add wiringpi support for RPI3

### PR DESCRIPTION
**Pull request by NicoHood**
Fixes:
```
Oops - unable to determine board type... model: 8
```

### CODE REVIEW

File: alarm/wiringpi/PKGBUILD [hunk 1]
```diff
@@ -5,10 +5,10 @@ buildarch=20
 
 _pkgname='wiringPi'
 pkgname=wiringpi
-pkgver=97.78b5c32
+pkgver=100.b0a60c3
 pkgrel=1
 pkgdesc="WiringPi: An implementation of most of the Arduino Wiring functions for the Raspberry Pi"
-arch=('arm')
+arch=('arm' 'arm7h')
```
Comment (kmihelich) [inline alarm/wiringpi/PKGBUILD]: What is arm7h?

Comment (NicoHood) [inline alarm/wiringpi/PKGBUILD]: I tried to rebuild the package on my raspi3 and it gave me an error, that the architecture is not supported for some reason. Thatswhy I added arm7h (arm7 hardfloat). No Idea how you could build it without this. You can build the package for arm6h though and run it on arm7, but then its slower.

Comment (kmihelich) [inline alarm/wiringpi/PKGBUILD]: ARM7 is not a supported architecture.

Comment (NicoHood) [inline alarm/wiringpi/PKGBUILD]: Supported of what? Wiringpi does support rpi3 (aka arm7) and alarm too!?

Comment (WarheadsSE) [inline alarm/wiringpi/PKGBUILD]: https://github.com/archlinuxarm/PKGBUILDs/blob/master/README.md#buildarch

man makepkg ; see -A

**Comment by NicoHood** [comment]
Ah I see. I removed it. It now should work.

BTW lirc does not work and gives me a buffer overflow. I checked back on raspbian which works fine. I cannot find a lirc PKGBUILD here, so I think its pulled from upstream. Could you maybe enable issues here to track bugs like this?

**Comment by kmihelich** [comment]
Seriously? 
https://github.com/archlinuxarm/PKGBUILDs/blob/master/CONTRIBUTING.md
Read it now, conform to it, or close this pull request and stop wasting my time.

**Comment by NicoHood** [comment]
You can squash the commits on github too (in the merge menu).

**Comment by NicoHood** [comment]
I've updated my branch and now it only uses a single commit. Will you reopen or do I need to create a new PR (which the guidelines do not want for sure)? I am sorry for the inconvenience this caused.

**Pull request closed without merging**